extension/app/ts/components/pages/InterceptorAccessList.tsx

```diff
@@ -195,12 +238,12 @@ export function InterceptorAccessList(param: InterceptorAccessListParams) {
 										{ access.addressAccess.length === 0 ? <p className = 'paragraph'> No individual address accesses given </p> : <>
 											{ access.addressAccessModified.map( (websiteAccessAddress, addressIndex) => (
 												<li style = { `margin: 0px; margin-bottom: ${ addressIndex < access.addressAccessModified.length - 1  ? '10px;' : '0px' }` }>
-													{ websiteAccessAddress.removed ? <p style = 'color: var(--negative-color)' > { `Forgot ${ websiteAccessAddress.address }`} </p> :
+													{ websiteAccessAddress.removed ? <p style = 'color: var(--negative-color)' > { `Forgot ${ ethers.utils.getAddress(addressString(websiteAccessAddress.address)) }`} </p> :
```

Comment:
Why are we doing bigint => string => string?  Is it because we ultimately need a checksummed string and there is no method for direct from bigint to checksummed string?  If so, then perhaps there is value in creating such a helper as it seems like a common need?

Reply:
yeah thats the reason, yeah, we could create helper for it